src/Text/Parsing/StringParser/String.purs

```diff
@@ -15,17 +15,22 @@ module Text.Parsing.StringParser.String
   , upperCaseChar
   , anyLetter
   , alphaNum
+  , regex'
+  , regex
   ) where
 
 import Prelude
 
 import Control.Alt ((<|>))
-import Data.Array ((..))
+import Data.Array ((..), uncons)
 import Data.Char (toCharCode)
 import Data.Either (Either(..))
 import Data.Foldable (class Foldable, foldMap, elem, notElem)
-import Data.Maybe (Maybe(..))
-import Data.String (Pattern(..), charAt, length, indexOf', singleton)
+import Data.Maybe (Maybe(..), fromMaybe)
+import Data.String (Pattern(..), charAt, drop, length, indexOf', singleton)
+import Data.String.Utils (startsWith)
```

Comment:
You could use `Data.String.stripPrefix` here, and avoid the extra dependency (which would necessitate a major version bump).